backend/aurcache/src/builder/docker.rs

```diff
@@ -120,7 +120,7 @@ impl Builder {
 
         let build_flags = self.package_model.build_flags.get()?.split(";").join(" ");
         // create new docker container for current build
-        let build_dir_base = "/var/cache/makepkg/pkg";
+        let build_dir_base = "/build";
```

Comment:
The actual build location doesn't really matter, but `/var/cache/makepkg/pkg` is a bit obscure when we just want _some_ working directory. It's pretty common to use top-level directories like `/build`, `/app`, ... But we can obviously use something else if you prefer.